File: backend/SalonManagement.API/DTOs/Employee.cs
namespace SalonManagement.API.DTOs.Employee
{
    public class EmployeeScheduleRequestDto
    {
        public int DayOfWeek { get; set; } // 0..6
        public TimeSpan StartTime { get; set; }
        public TimeSpan EndTime { get; set; }
    }

    public class EmployeeServiceAssignDto
    {
        public Guid ServiceId { get; set; }
    }
}

File: frontend/src/api/employee.ts
import axiosInstance from './axiosConfig';

export const employeeApi = {
  getSalonWorkingHours: () => axiosInstance.get('/employee/working-hours'),
  getMySchedules: () => axiosInstance.get('/employee/schedules'),
  addSchedule: (data: { dayOfWeek: number; startTime: string; endTime: string }) => axiosInstance.post('/employee/schedules', data),
  //updateSchedule: (id: string, data: { dayOfWeek: number; startTime: string; endTime: string }) => axiosInstance.put(`/employee/schedules/${id}`, data),
  deleteSchedule: (id: string) => axiosInstance.delete(`/employee/schedules/${id}`),

  getSalonServices: () => axiosInstance.get('/employee/services'),
  getMyServices: () => axiosInstance.get('/employee/my-services'),
  assignService: (data: { serviceId: string }) => axiosInstance.post('/employee/my-services', data),
  removeMyService: (id: string) => axiosInstance.delete(`/employee/my-services/${id}`)
};
